.github/workflows/build.yml

```diff
@@ -373,8 +373,14 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ needs.set-tags.outputs.git_ref }}
-      - run: |
-          mkdir -p target/release
+      - name: Create local folders
+        run: |
+          mkdir -p target/release/wbuild/${{ matrix.chain }}-runtime/
+      - name: "Download branch built runtime"
+        uses: actions/download-artifact@v4
+        with:
+          name: runtimes
+          path: target/release/wbuild/${{ matrix.chain }}-runtime/
```

Comment:
This is already done in the next step

Reply:
@RomarQ no it was done for zombi tests but not for dev tests. I added a dev-test that need access to the wasm runtime artifacts, so we need to download them inside dev-test job